source/fundamentals/connection/connect.txt

```diff
@@ -77,12 +77,36 @@ In the example, we set two connection options: ``maxPoolSize=20`` and
 
 .. _connect-atlas-java-driver:
 
-The following code shows how you can use the sample connection URI in a client
-to connect to MongoDB.
+Atlas Connection Example
+------------------------
+
+To connect to a MongoDB deployment on Atlas, create a client. You can
+create a client that uses your connection string and other 
+client options by passing a ``MongoClientSettings`` object to the
+``MongoClients.create()`` method.
+
+To instantiate a ``MongoClientSettings`` object, call the
+``MongoClientSettings.builder()`` method.
+
+You can chain methods to your ``MongoClientSettings`` instance to specify
+your connection string and other client options. Call the
+``applyConnectionString()`` method on your ``MongoClientSettings``
+instance to specify your connection URI. You must pass your connection
+URI to the ``ConnectionString`` constructor to create a valid parameter
+for the ``applyConnectionString()`` method.
```

Comment:
Suggestion:

I think the action "[creating] a valid parameter" should instead describe the accepted type. When I read "valid parameter", I assumed that there might be multiple types or ranges of values that could qualify it as a "valid parameter". If it's easier, maybe being less specific about methods and types could simplify this.

E.g. 
```suggestion
instance to specify your connection URI. 
```